### PR TITLE
Fix logo height

### LINKED ISSUE #476: Updating logo nav component in header
## Problem
In staging, Openverse logo and symbol are used in the Header component, but there are inconsistent sizes and focus styles. I think this happens because the root component is not the same, as `VBrand` and `VLogoLoader` do not relate to each other. See the following screenshots from staging environment.

### Header
|Resting|Focus|
|:---|:---|
|CleanShot 2022-12-26 at 11 26 37@2x|CleanShot 2022-12-26 at 11 26 57@2x|

### Header internal
|Resting|Focus|
|:---|:---|
|CleanShot 2022-12-26 at 11 27 15@2x|CleanShot 2022-12-26 at 11 27 25@2x|

Now that we started working on the homepage redesign (#1433) it seems worth improving this key nav element.

_Edited version: This update does not affect the Footer component as it uses `VBrand` for the logo._

## Description

Upon the homepage redesign, [Logo nav component](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=3835%3A11055&t=HOhJ0clMNscVFnvS-4) was updated in Figma to simplify the versions used across the site. Including the homepage and the full-screen menu.

The component has three sizes:
* `Small` for the header's small breakpoints where Openverse symbol lives inside the search bar.
* `Medium` for the header's large breakpoints where Openverse symbol lives outside the search bar.
*  `Large` for the header and footer versions where Openverse logo is displayed completely. 

And three states: `Resting`, `Hover`, `Focus`, and `Loading`. The last one is for `Small` and `Medium` sizes as those sizes are used in the search flow.

## Additional context
This request impacts the subsequent work coming from WordPress/openverse-frontend#1433 as
* WordPress/openverse-frontend#2051
* WordPress/openverse-frontend#2060

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Fixes #476 by @fcoveram.

## Description

Fixes the logo by reducing its size and making its size universal. Also removes "baked in" padding to the logo loader; this instead comes from the context of the loader. 

The Openverse logo should be the same size everywhere, now.

This PR also removes the global `maxDiffPixelRatio` from our Playwright tests. These tests are fortunately much more stable these days, and removing this gives us better assurances we are capturing changes to our front-end. 

For example, with that setting enabled, our tests missed the fact that the "Terms" link was added to the header! Some snapshot changes only reflect the change in this link and not a change to the logo.

## Testing Instructions

Run the PR locally with `just frontend`. The Openverse logo should be the same size everywhere, now. Click through the site and confirm. Tests should pass too.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [x] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by obulat — DISMISSED** [review]
The `maxDiffPixelRatio` of 0.01 does not catch the small changes in the header. We can lower it to 0.005 to prevent extreme flakiness while also catching differences in the logos.

My request for change is to lower the ratio to 0.005 and update the snapshots:
https://github.com/WordPress/openverse/blob/6b49eed05b34568e439795bd86f844268b480985/frontend/test/storybook/playwright.config.ts#L19
 
I think you will also need to rebase onto main because there were some updates to snapshots in recently-merged PRs.

File: frontend/src/components/VLogoLoader/VLogoLoader.vue [hunk 1]
```diff
@@ -4,13 +4,11 @@
...
     :class="{
       loading: status === 'loading' && !prefersReducedMotion,
-      'h-10 w-10': autoResize,
-      'h-12 w-12': !autoResize,
     }"
     aria-hidden="true"
     :data-prefers-reduced-motion="prefersReducedMotion"
     data-testid="logo-loader"
-    class="inline-flex items-center justify-center rounded p-3 md:h-12 md:w-12"
+    class="inline-flex h-[18px] items-center justify-center rounded"
```
Comment (fcoveram) [inline frontend/src/components/VLogoLoader/VLogoLoader.vue:11]: 18px height looks better than the small size variant in the Design Library. I will update the figma library to mirror the change. Nice ✨

**Review by fcoveram — APPROVED** [review]
Everything looks correct 🚀

**Comment by zackkrida** [comment]
@obulat of course, the maxDiffPixelRatio! Thank you, I will play with this a little.

**Comment by zackkrida** [comment]
This one is going to take a little bit of time and effort to dial in the tests and identify any existing flake with the pixel ratio.

**Review by obulat — APPROVED** [review]
Only 93 files 🎉 Great that no other snapshots had to be updated.

Would be nice to make sure that this PR is up-to-date before merging to avoid test regressions.

**Pull request merged**